packages/aragon-rpc-messenger/src/index.js

```diff
@@ -91,7 +91,14 @@ export default class Messenger {
     const id = this.send(method, params)
 
     return this.responses().pipe(
-      filter((message) => message.id === id)
+      filter((message) => message.id === id),
+      map((response) => {
+        if (response.error) {
```

Comment:
Is there any reason not to leave the `response.error` an error instance in https://github.com/aragon/aragon.js/blob/master/packages/aragon-rpc-messenger/src/jsonrpc.js#L19?

It seems like we are converting an error to a string in jsonrpc.js and back to an error here.

Reply:
Primarily because sending an `Error` instance through `postMessage()` will fail: [see things that don't work with the structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#Things_that_don't_work_with_structured_clone).